README.rst

```diff
@@ -111,9 +111,9 @@ again. So, if you use ``aiorun`` this is what **you** need to remember:
   will have ``CancelledError`` raised internally. It's up to you whether
   you want to handle this inside each coroutine with
   a ``try/except`` or not.
-- If you want to protect coros from cancellation, see `shutdown_waits_for()`
+- If you want to protect corals from cancellation, see `shutdown_waits_for()`
```

Comment:
This suggestion is incorrect. I am starting to suspect that you used some kind of automation to make these changes, but many of the suggestions are wrong. Did you check these changes yourself?